kyaml/fn/framework/framework.go

```diff
@@ -256,6 +256,9 @@ type TemplateCommand struct {
 	// Templates is a list of templates to render.
 	Templates []*template.Template
 
+	// PatchTemplates is a list of templates to render into Patches and apply.
+	PatchTemplates []PatchTemplate
```

Comment:
This is going to be interesting :)

Reply:
It lets you do some crazy stuff ;)

Reply:
I'm using this to port the kubebuilder kustomize configuration to a kustomize function which generates the otherwise skaffolded config.